Jenkinsfile

```diff
@@ -24,7 +24,11 @@ ircMsgResult(CHANNELS) {
 
     // Runs `make itest_${version}` and attempts to upload to apt server if not an automatically timed run
     // This will automatically break all the steps into stages for you
-    debItestUpload(PACKAGE_NAME, DIST)
+    debItestUpload(
+        repo: PACKAGE_NAME,
+        versions: DIST,
+        committish: commit,
```

Comment:
does groovy allow trailing commas?

Reply:
groovy playground seems to not complain about trailing commas on a `print` call.

Reply:
it does :)